munit/shared/src/main/scala/munit/Assertions.scala

```diff
@@ -120,6 +120,40 @@ trait Assertions extends MacroCompat.CompileErrorMacro {
     }
   }
 
+  /**
+   * Asserts that two doubles or floats are equal to within a positive delta.
+   * If the expected value is infinity then the delta value is ignored.
+   * NaNs are considered equal: assertEquals(Double.NaN, Double.NaN, *) passes.
+   */
+  def assertEquals(
```

Comment:
What do you think about renaming this method?

```suggestion
  def assertEqualsWithDelta(
```

It would be nice to avoid method overloads, currently there is only one `assertEquals`.

Reply:
How about `assertEqualsDouble` (and maybe `assertEqualsFloat`)? The reasoning being that you would basically never want to check the equality of two floating point numbers without using a delta.

Reply:
Sounds good to me.